src/components/learning-center/article-footer.tsx

```diff
@@ -44,8 +43,16 @@ const TableOfContentsSection = (props: TableOfContentsSection) => {
   );
 };
 
-const FooterCta = (props: any) => {
-  const localePrefix = props.locale ? "/" + props.locale : "";
+type FooterCtaProps = {
+  content: {
+    title: string;
+    subtitle: string;
+    ctaLink: string;
+    ctaText: string;
+  };
+};
```

Comment:
I like this approach of slowly adding in more explicit type definitions as we go.